### PR TITLE
Cleanup some deprecated, unused allows

### PR DESCRIPTION
**Pull request by klensy**
Probably need squash.

### CODE REVIEW

**Comment by klensy** [comment]
Well, looks like *something* not unused.

File: src/windows/setup_config.rs [hunk 1]
```diff
@@ -23,22 +19,9 @@ use std::{
...
-pub const eComplete: InstanceState = -1i32 as u32;
 
 RIDL! {#[uuid(0xb41463c3, 0x8866, 0x43b5, 0xbc, 0x33, 0x2b, 0x06, 0x76, 0xf7, 0xf4, 0x2e)]
 interface ISetupInstance(ISetupInstanceVtbl): IUnknown(IUnknownVtbl) {
-    fn GetInstanceId(
-        pbstrInstanceId: *mut BSTR,
-    ) -> HRESULT,
-    fn GetInstallDate(
-        pInstallDate: LPFILETIME,
-    ) -> HRESULT,
```
Comment (ChrisDenton) [inline src/windows/setup_config.rs:41]: By deleting these. you're changing the layout of the vtable.

Comment (klensy) [reply src/windows/setup_config.rs]: Yep, this commit broken tests.

**Review by NobodyXu — APPROVED** [review]
Thanks!

**Pull request merged**